InternetShop/InternetShop/ReadUser.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using InternetShop.Controllers;
+using Npgsql;
+
+namespace InternetShop
+{
+    class DataReader
```

Comment:
What access modifier this?

Reply:
usually we need to write access modifier

Reply:
f

Reply:
done

Reply:
done